charts/mapfish-print/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
 name: mapfish-print
 description: Helm chart for MapFish Print v3
-version: 1.0.0
-appVersion: 3.29.3
+version: 1.1.0
+appVersion: 3.30.5
```

Comment:
Are any incompatibilities expected in config in this bigger version update?

Reply:
I don't really expect this. But as the change in the appVersion is on the feature level, i also set the new chart version on this level.

I don't know what is best practice here :shrug: